apps/docs/stories/components/PaymentForm/index.stories.tsx

```diff
@@ -118,23 +123,16 @@ const meta: Meta = {
     },
   },
   decorators: [
-    // @ts-ignore
-    withActions,
+    withActions
   ],
 };
 
 const Template = (args: any) => {
   // The <div> here should be replaced by a `display` property in the cardForm potentially
   return `
-    <div>
-      <justifi-payment-form
-        ${getAttributesString(args)}
-      />
-    </div>
+    <justifi-payment-form ${getAttributesString(args)}></justifi-payment-form>
```

Comment:
@jakemer not sure where the issue is in this file - but I'm seeing the new Theme selector that works exactly as the Checkout story, and I'm also seeing an additional select control called `theme` (lowercase) that seems to be an accidental addition. This additional control doesn't have the same options, and doesn't actually change anything. But we should fix that. 

But otherwise the dark and light themes look good on the PaymentForm. 👍🏻